alacritty/src/event.rs

```diff
@@ -1149,6 +1149,7 @@ impl<N: Notify + OnResize> Processor<N> {
                     display_update_pending.set_dimensions(PhysicalSize::new(width, height));
 
                     processor.ctx.window_mut().dpr = scale_factor;
+                    info!("Device pixel ratio changed: {}", scale_factor);
```

Comment:
I added this `info!` log message to the `DPRChanged` event so that you can look at the info logs and get a complete sense of what has happened to the DPR. Currently, we just log the estimated and initial DPR, so now it'll be clear when things update as well.

Reply:
That should already be logged [here](https://github.com/alacritty/alacritty/blob/master/alacritty/src/renderer/mod.rs#L346) though, right?

Reply:
Ah, right you are. Should be hit every time the event is, so I'll remove the added log.